pkg/app/application.go

```diff
@@ -114,7 +114,7 @@ func (a *Application) resolveRefsToGeneratedResources(m resmap.ResMap) (resmap.R
 	}
 	r = append(r, t)
 
-	err = transformers.NewMultiTransformer(r).Transform(m)
+	err = transformers.NewMultiTransformer(r, false).Transform(m)
```

Comment:
Sorry, could you just leave this signature alone and add a new factory method like
NewMultiTransformerWithConflictCheck
https://www.codereadability.com/boolean-arguments